fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  /* Implement useEffect to fetch all todos. Hints are at the bottom of the file. */
+  useEffect(() => {
+    const getTodos = async () => {
+      try {
+        const res = await fetch('/api/todos');
+        if (!res.ok) {
+          throw new Error('Fetch Error: ', res.status);
+        }
+        const todos = await res.json();
+        setTodos(todos);
+      } catch (err) {
+        setError(err);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+    getTodos();
+  }, [todos]);
+
+  /* Implement addTodo to add a new todo. Hints are at the bottom of the file. */
+  async function addTodo(newTodo) {
+    const addTodo = async () => {
```

Comment:
Since this is in a handler and not a `useEffect`, you don't have to use the "define async function/call it" strategy. You can just write the code directly. 
Note that handlers can be async. React properly manages async handlers.